job_board/models/preference.py

```diff
@@ -1,13 +1,34 @@
 from django.db import models
 from .job_type import JobType
 from .location import Location
-from .years_of_experience import YearsOfExperience
 
 
 class Preference(models.Model):
+
+    class yearsOfExperience(models.TextChoices):
+        NOEXPERIENCE = "no experience", "no experience"
+        YEARS0TO2 = "0-2 years", "0-2 years"
```

Comment:
Just a question, why do you write every description twice?

Reply:
Please rebase and push

Reply:
@paOmer This is a type of enum Django support.
Each enum here is a two-tuple (value, label).
value - the actual value.
label - a more human-readable name

In my case, the value is readable so it is also the label.
A good example will be: RED = "#FF0000", "Red"

You can read more here: https://docs.djangoproject.com/en/4.0/ref/models/fields/#field-choices-enum-types

Reply:
so you you could suggest some thing as follow:
NOEXPERIENCE = "no experience",
etc

Reply:
@DeanBiton ^^

Reply:
@Yarboa Currently I am working on a ModelForm based on the Preference model and it seems that changing the Enum to NOEXPERIENCE = "no experience" changed the option it showed me on the webpage from "no experience" to "NOEXPERIENCE ".

I tried changing this for a while with no success, should I revert the enum back to NOEXPERIENCE = "no experience", "no experience"?